CODEOWNERS

```diff
@@ -73,6 +73,7 @@ drivers/serial/uart_altera_jtag.c        @andrewboie
 drivers/serial/uart_riscv_qemu.c         @fractalclone
 drivers/slip/*                           @jukkar @tbursztyka
 drivers/spi/*                            @tbursztyka
+drivers/spi/spi_ll_stm32.*               @mbolivar
```

Comment:
Do you want to add yourselves? There's no issue in doing so -- I just didn't want to "volunteer" you for future review.